requirements/testing_sheet.txt

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Is there a rationale behind putting this in the Kolibri repo's `requirements` folder? Seems to me this is pretty independent of the Kolibri project. Might be better to just bake these into the `Dockerfile`

@indirectlylit thoughts on this?

Reply:
The requirements folder has a lot of 'meta' stuff in it so I don't see any particular issue.

Also the integration gherkin files are in fact part of the Kolibri project - they're checked into the root directory. To me this feels similar to the docs building, testing, and font management requirements.

I think the main question is whether these are _only_ used from within Docker, or if it's also possible to run the migration script outside of docker?

Reply:
It's also possible to run the script outside the docker environment. Running a docker file in the local machine take more time than execute the script directly.

Reply:
I'd like to add that it's only slowness is largely due to the current implementation. Caching could make it just as fast as a native run. I do doubt, though, that it could get any faster than a native run, since the script relies on the GitHub-hosted repo